amethyst_core/src/transform/components/local_transform.rs

```diff
@@ -252,3 +264,60 @@ impl CgTransform<Point3<f32>> for Transform {
         }
     }
 }
+
+impl CgTransform<Point2<f32>> for Transform {
+    fn one() -> Self {
+        Default::default()
+    }
+
+    fn look_at(_eye: Point2<f32>, _center: Point2<f32>, _up: Vector2<f32>) -> Self {
```

Comment:
Wouldn't that just be a rotation?
Also I'd prefer if _center was called target or something, if this method could be made to rotate.
I guess you'd need a forward axis too.

Also, can't you use compile_error! for that?

Example code:
```
float rot_z = Mathf.Atan2(diff.y, diff.x) * Mathf.Rad2Deg;
transform.rotation = Quaternion.Euler(0f, 0f, rot_z - 90);
```

Reply:
What I'm saying is that you can compute a look_at for 2d. You just need to set the up vector to the perpendicular of the direction vector.

Reply:
What I'm getting at is that it makes very little sense to have a look_at for 2D. Yes, you could make it just apply a rotation based on the up vector. I just felt like it wasn't really gonna be used by anyone, since there are better functions that apply rotation directly that are much more obvious.